docs/content/preview/develop/realworld-apps/ecommerce-app.md

```diff
@@ -130,7 +133,7 @@ insert_batch.push({
 
 The dynamic attributes are loaded using the Redis-compatible YEDIS API into YugabyteDB.
 
-```
+```yedis
```

Comment:
Does this actually produce useful source-highlighting? If not, select from [one of these](https://gohugo.io/content-management/syntax-highlighting/#list-of-chroma-highlighting-languages).

Edit: looks like Redis themselves don't even bother with syntax highlighting, so I'd just go with `text`.